battle_test.go

```diff
@@ -6,6 +6,8 @@ import (
 	. "github.com/onsi/gomega"
 )
 
+var defaultMoveOpt = WithMoves(GetMove(MovePound))
```

Comment:
Do we want to have some kind of no-op move as a default in `GeneratePokemon` instead? I feel like most cases for using pound are to create a valid Pokemon with 1 move that serves no purpose.

Edit: I see the new issue for this now.